shared/haml/hamburger.haml

```diff
@@ -82,7 +103,7 @@
   end
 
   # Do we show hamburger on all widths, only mobile, or not at all?
-  if show_teacher_options == "show-always" || show_student_options == "show-always" || show_pegasus_options == "show-always"
+  if show_teacher_options == "show-always" || show_student_options == "show-always" || show_pegasus_options == "show-always" || language != "en"
```

Comment:
Presumably we can work out what category we're showing for non-en users and use that.